lib/src/marker_cluster_layer.dart

```diff
@@ -430,49 +430,49 @@ class _MarkerClusterLayerState extends State<MarkerClusterLayer>
 
     if (layer is MarkerNode) {
       if (!_boundsContainsMarker(layer)) {
-        return List<Widget>();
+        return <Widget>[];
       }
 
       // fade in if
       // animating and
       // zoom in and parent has the previous zoom
-      if (_zoomController.isAnimating &&
-          (_currentZoom > _previousZoom &&
-              layer.parent.zoom == _previousZoom)) {
+      if (_zoomController!.isAnimating &&
+          (_currentZoom! > _previousZoom! &&
+              layer.parent!.zoom == _previousZoom)) {
         // marker
         layers.add(_buildMarker(
             layer,
-            _zoomController,
+            _zoomController!,
             FadeType.FadeIn,
             TranslateType.FromNewPosToMyPos,
-            _getPixelFromMarker(layer, layer.parent.point)));
+            _getPixelFromMarker(layer, layer.parent!.point)));
         //parent
-        layers.add(_buildCluster(layer.parent, FadeType.FadeOut));
+        layers.add(_buildCluster(layer.parent!, FadeType.FadeOut));
       } else {
-        layers.add(_buildMarker(layer, _zoomController));
+        layers.add(_buildMarker(layer, _zoomController!));
       }
     }
     if (layer is MarkerClusterNode) {
-      if (!_boundsContainsCluster(layer)) {
-        return List<Widget>();
+      if (_boundsContainsCluster(layer)) {
```

Comment:
I think you accidentally changed this condition and it causes the clustering icons to not appear.

Reply:
good catch, reverted back